### DISCUSSION TITLE
Installation Fails magick wizard: wizard.jpg  unable to open image 'wizard:'

### DISCUSSION THREAD

**Discussion by letgluedry** [ImageMagick/ImageMagick#6504] [Help]
### ImageMagick version

7.1.1-13

### Operating system, version and so on

Windows 7 Home Premium 64-bit

### Description

Using ImageMagick-7.1.1-13-Q16-HDRI-x64
After installing, the commands suggested by the installer failed.
Does this matter?
Any thoughts on what to do?

...>magick wizard: wizard.jpg
magick: unable to open image 'wizard:': Invalid argument @ error/blob.c/OpenBlob/3569.
magick: no decode delegate for this image format `' @ error/constitute.c/ReadImage/746.

...>magick logo: logo.gif
magick: unable to open image 'logo:': Invalid argument @ error/blob.c/OpenBlob/3569.
magick: no decode delegate for this image format `' @ error/constitute.c/ReadImage/746.

I tried entering the full path to wizard.jpg:
Q:\barn\scratch>magick wizard: "C:\Program Files\ImageMagick-7.1.1-Q16-HDRI\images\wizard.jpg"
magick: unable to open image 'wizard:': Invalid argument @ error/blob.c/OpenBlob/3569.
magick: no decode delegate for this image format `' @ error/constitute.c/ReadImage/746.

**Comment by fmw42** [comment]
What do you get from 

`magick -version`

**Reply by letgluedry** [reply]
...>magick -version
Version: ImageMagick 7.1.1-13 Q16-HDRI x64 d5974ce:20230716 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png ps raqm raw rsv
g tiff webp xml zip zlib
Compiler: Visual Studio 2022 (193532217)

Also, these are installed:
Microsoft Visual C++ 2015-2022 Redistributable (X64) - 14.34.31931
Microsoft Visual C++ 2015-2022 Redistributable (X86) - 14.32.31326

**Reply by fmw42** [reply]
Looks OK to me.  But a Windows user will need to help further.  How did you install?  Did you use instructions from https://imagemagick.org/script/download.php#windows

**Reply by letgluedry** [reply]
Yes, I followed those instructions.
And several versions of vcomp140.dll are in 
C:\Windows\SysWOW64\
C:\Windows\System32\
C:\Program Files\ImageMagick-7.1.1-Q16-HDRI\

**Comment by snibgo** [comment]
The problem may be:
> Windows 7 Home Premium 64-bit

I think I remember a developer saying that Windows 7 was no longer supported. But I could be wrong.

**Reply by letgluedry** [reply]
From  https://imagemagick.org/script/download.php#windows
"ImageMagick runs on Windows 7 (x86 , x64 & arm64) or newer, ...".
But that page's copyright is 1999, so maybe the developers have raced ahead of the documenters.

**Comment by snibgo** [comment]
I can only guess at the problem. I don't have a Windows 7 computer.

How old is your computer? How much RAM does it have?

You have installed a `x64` version, so I assume you have a x64 processor (as apposed to x86 or arm64)?

You might try installing a portable version without HDRI, for example https://imagemagick.org/archive/binaries/ImageMagick-7.1.1-13-portable-Q16-x64.zip

**Reply by letgluedry** [reply]
The CPU is from 2012.  The motherboard is from 2015.
16 GB
From Belarc:
3.30 gigahertz Intel Core i3-2120
64-bit ready
Multi-core (2 total)
Hyper-threaded (4 total)
When I run (double-click) the portable's magick.exe, what looks like a command window flashes on the screen, displaying what looks like an error message.  I can see "invalid argument".

**Comment by letgluedry** [comment]
When I run Magick by double-clicking the desktop icon, and then opening a JPG, this appears:
![Capture](https://github.com/ImageMagick/ImageMagick/assets/140099034/0f16e8fe-c8ca-4a6b-8655-3b32ca8c0a5b)

**Comment by snibgo** [comment]
> When I run (double-click) the portable's magick.exe, what looks like a command window flashes on the screen, displaying what looks like an error message. I can see "invalid argument".

Don't do that. `magick` is a command-line program. You should start a console (aka "dos window") and type commands such as:
```
magick -version

magick wizard: x.png
```

**Comment by JimLikes** [comment]
I'm a little late in the game, but in my case it looks like it was a permissions problem on C:\Program files\ImageMagick-7.1.1-Q16-HDRI\Images.    When I moved the image to my user area or chose "run as administrator' when I launched CMD then the error went away.

**Comment by rmathar** [comment]
I have the same error message with ImageMagick-7.1.1-21 Q16-HDRI x86_64 21667. It turns out that image files which are symbolic links are no longer recognized on Linux systems. Converting the file that they point to removes the error. This seems to be a retraction from earlier ImageMagick Versions (as I remember that this worked a month ago on the same computer under openSUSE 15.6)
